lib/bosh/workspace/helpers/stemcell_helper.rb

```diff
@@ -36,10 +39,38 @@ def project_deployment_stemcells
       end
     end
 
+    def download_stemcell_from_bosh_io(stemcell)
+
+      url=sprintf("https://bosh.io/d/stemcells/%s?v=%s", stemcell.name, stemcell.version)
```

Comment:
Could you tell why do you use this notation? Isn't `url = "https://bosh.io/d/stemcells/#{stemcell.name}?v=#{stemcell.version}"` more ruby friendly?